libraries/botbuilder/tests/streaming/botFrameworkAdapterStreaming.test.js

```diff
@@ -298,53 +222,48 @@ describe('BotFrameworkAdapter Streaming tests', () => {
             expect(response.statusCode).to.equal(501);
         });
 
-        it('returns a 500 when bot can not run', async () => {
-            const MiddleWare = require('botbuilder-core');
-            const bot = {};
-            let mw = {
+        it('returns a 500 when BotFrameworkAdapter.logic is not callable', async () => {
+            const adapter = new BotFrameworkAdapter();
+            const request = new MockStreamingRequest();
+
+            const response = await adapter.processRequest(request);
+            expect(response.statusCode).to.equal(500);
+        });
+
+        it('returns a 500 and calls middleware when BotFrameworkAdapter.logic is not callable', async () => {
```

Comment:
BotAdapter.runMiddleware will run all middleware before attempting to call the application logic. This test could be stored in the previous test, but I separated the two to make the tests more explicit and preserve the "intent" of the previous test.